posts/metriq_release.html

```diff
@@ -0,0 +1,198 @@
+            width: 100%;
+        }
+
+        #mce-EMAIL {
+            padding: 11px;
+            margin: 10px;
+        }
+
+        .logo-image {
+            display: inline-block;
+            width: 140px;
+            height: auto;
+            max-width: 180px;
+            max-height: 180px;
+            padding: 12px;
+        }
+    </style>
+
+    <!-- Global site tag (gtag.js) - Google Analytics -->
+    <script async src="https://www.googletagmanager.com/gtag/js?id=UA-19932157-4"></script>
+    <script>
+        window.dataLayer = window.dataLayer || [];
+
+        function gtag() {
+            dataLayer.push(arguments);
+        }
+        gtag('js', new Date());
+
+        gtag('config', 'UA-19932157-4');
+    </script>
+    <title>QuTiP 2021 Annual Review</title>
+</head>
+
+<body>
+    <div class="layout">
+        <main id="content" class="content">
+            <section class="heavy">
+                <div class="container">
+                    <div class="col-3">
+                        <div class="hero">
+                            <a href="/"><img src="/logos/logov3.svg" alt="Unitary Fund" /></a>
+                            <p class="tagline"><b>Because evolution is
+                                    <a href="https://en.wikipedia.org/wiki/Unitarity_(physics)"
+                                        target="_blank">unitary</a></b>.</p>
+                            <button id="mobile-menu-icon" class="hamburger hamburger--squeeze" type="button">
+                                <span class="hamburger-box">
+                                    <span class="hamburger-inner"></span>
+                                </span>
+                            </button>
+                        </div>
+                        <section class="light">
+
+
+                            <right>
+                                <h2 class="leading-arrow">Unitary Fund launches Metriq, a platform for community driven quantum benchmarks </h2>
+                                <h3></h3>
+                            </right>
+                            2022-05-17
+                            <br> Dan Strano
+
+
+                            <div>
+                                <p>Unitary Fund is proud to announce a new open source software tool for the quantum computing community, at <a href="https://metriq.info">https://metriq.info</a>!</p>
+                                <p>With the growth in quantum computing in academia and industry, it can be hard to keep track of the many new developments. Metriq helps with this. Any user can (1) make <a href="https://metriq.info">submissions</a> that show performance of <a href="https://metriq.info/Methods">methods</a> against <a href="https://metriq.info/Tasks">tasks</a> and/or (2) define new tasks that results can be submitted against. For example see historical results for <a href="https://metriq.info/Task/34">quantum volume</a>, a task commonly used to benchmark hardware performance.</p>
```

Comment:
The current "that" word sounds strange to me but please keep it if it is correct.
```suggestion
                                <p>With the growth in quantum computing in academia and industry, it can be hard to keep track of the many new developments. Metriq helps with this. Any user can (1) make <a href="https://metriq.info">submissions</a> that show performance of <a href="https://metriq.info/Methods">methods</a> against <a href="https://metriq.info/Tasks">tasks</a> and/or (2) define new tasks whose results can be submitted against. For example see historical results for <a href="https://metriq.info/Task/34">quantum volume</a>, a task commonly used to benchmark hardware performance.</p>
```

Reply:
Your other change sounded right to me, but this one doesn't. Sorry, I'll accept the other, but I think this one should stay as-is.

Reply:
:+1: